packages/babel-parser/src/parser/util.js

```diff
@@ -407,4 +415,5 @@ export default class UtilParser extends Tokenizer {
 export class ExpressionErrors {
   shorthandAssign = -1;
   doubleProto = -1;
+  optionalParameters = -1;
```

Comment:
Can you add usage comments about `optionalParameters` above?

Reply:
Oh I meant for the usage of `optionalParameters`. What does `optionalParameters` do? In which plugin does it serve? Anything contributors should be pay attention? Like we did for `shorthandAssign` and `doubleProto` above.